### ISSUE TITLE
The Debian packaging should be in the git repo

### ISSUE THREAD

**Issue by marmistrz** [shashlik/shashlik-build#20]
It would make much easier for the people to build Shashlik on their own, especially that installing is not really documented.

**Comment by mdeguzis** [comment]
Exactly, the static page only lists the deb, not the debian tarball. I don't expect the full orig.tar.gz to be there, but upload the debian folder, _please_. Instructions at http://www.shashlik.io/what-is/ don't really work. See the latest comments.
